dev/io.openliberty.netty.internal/src/io/openliberty/netty/internal/tls/package-info.java

```diff
@@ -0,0 +1,12 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * 
```

Comment:
This new line in the copyright header isn't needed.

Reply:
I'll update all of these